Dockerfile

```diff
@@ -22,7 +22,9 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     libsasl2-dev \
     libzstd-dev \
     make \
-    pkg-config
+    pkg-config;
+    apt-get clean; \
+	rm -rf /var/lib/apt/lists/*;
```

Comment:
There's no need to run clean or clear apt lists since they are part of a mount type cache. That will just make the next build take longer